repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -0,0 +1,56 @@
+= Attribute Mapping Policy Validation Filter
+
+This filter will validate attribute mapping policies in requests to the origin service.
+
+The origin service is expected to be an Identity service.
+This filter will only take action if the HTTP request method is `PUT`.
+
+== General filter information
+* *Name:* attribute-mapping-policy-validation
+* *Released:* TBD
+* *Bundle:* repose-extensions-filter-bundle
+
+[NOTE]
+====
+This filter has no configuration.
+====
+
+== Prerequisites & Postconditions
+=== Required headers
+* `Content-Type` - Declares the content type of the policy (e.g., XML or JSON) which will be used during validation.
+
+=== Required preceding filters
+None.
+
+=== Request headers created
+None.
+
+=== Request body changes
+None.
+
+=== Recommended follow-on (succeeding) filters
+* <<saml-policy.adoc#,SAML Policy Translation Filter>> - These two filters work well together to provide mechanisms for attribute mapping policy management and application.
+
+=== Response body changes
+None.
+
+=== Response headers created
+None.
+
+=== Response Status Codes
+[cols="2", options="header,autowidth"]
+.Status Codes
+|===
+| Status Code
+| Reasons
+
+| `400`
```

Comment:
That seems really noisy to me. I would prefer the status codes be links with that info or maybe have tooltips (although a quick Google shows this isn't supported by Asciidoctor).